python/.flake8

```diff
@@ -1,7 +1,10 @@
 [flake8]
 max-line-length = 100
 exclude =
+  **/build/**,
   examples/protobuf/*pb2.py,
   mcap-protobuf-support/tests/*pb2.py,
   mcap-ros1-support/mcap_ros1/vendor/genpy/*.py,
   mcap-ros1-support/mcap_ros1/vendor/genmsg/*.py,
+  mcap-ros2-support/mcap_ros2/vendor/rosidl_adapter/*.py,
+ignore = E203 W503
```

Comment:
add a comment on why this is here? I assume this was the disagreement between black and flake8?

Reply:
More specifically, E203 violates PEP8. Can I put comments in this file? I can't tell what the syntax is

Reply:
Why disable them for the whole project?

Reply:
> Why disable them for the whole project?

Because E203 violates PEP8. It should not be enabled on any project. It shouldn't exist at all. One of the first things flake8's config documentation shows you is how to disable E203 (https://flake8.pycqa.org/en/latest/user/configuration.html).

The others are disabled across the whole project because they directly conflict with the black formatting we're applying everywhere.

Reply:
It looks like TOML. # comments probably work

Reply:
Does flake8 have a mode that tells it to ignore everything that black would take care of? Kind of like eslint-config-prettier, which disables the builtin eslint rules that would conflict or duplicate things prettier will change during formatting.